src/gsi/setuprad.f90

```diff
@@ -937,10 +957,72 @@ subroutine setuprad(obsLL,odiagLL,lunin,mype,aivals,stats,nchanl,nreal,nobs,&
           total_cloud_cover = tcc(1)
           cld = total_cloud_cover
         else
+
+!  If not allsky
+
+!  Generate model derived imager values for IASI or CrIS for CADS
+
+          if ((iasi_cads .and. iasi) .or. (cris_cads .and. cris)) then
+               call destroy_crtm
+               itmp1_cads = len(trim(obstype))
+               itmp2_cads = len(trim(isis))
+             if ( iasi ) then
+               isis_cads = 'avhrr3'//isis(itmp1_cads+1:itmp2_cads)
+               obstype_cads = 'avhrr'
+               nchanl_cads = 3   ! CRTM channels 3 - 5
+             elseif ( cris ) then
+!!               isis_cads = 'viirs-m'//isis(itmp1+1:itmp2)
```

Comment:
Remove comments?

Reply:
I removed the various comments except the last one (isis_cads = 'viirs-m'.......
This line will replace the following lines when the satellite id for viirs becomes consistent with cris.  Similar comments exist within read_cris.f90